src/poggit/ci/lint/PhpstanLint.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * Poggit
+ *
+ * Copyright (C) 2016-2020 Poggit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace poggit\ci\lint;
+
+use function htmlspecialchars;
+
+class PhpstanLint extends BuildLint {
+    public $level = BuildResult::LEVEL_WARN;
+
+    /** @var string|null */
+    public $file = null;
+    /** @var int */
+    public $line;
+    /** @var string */
+    public $message;
+
+    public function echoHtml() {
+        ?>
+      <h5>PHPStan</h5>
+      <p><?php if($this->file !== null){ ?>Problem found in <?= htmlspecialchars($this->file) ?> at line <?= $this->line ?>. :</p><?php } ?>
+      <pre class="code"><?= htmlspecialchars($this->message) ?></pre>
+        <?php
```

Comment:
PhpstanLint.php and PhpstanInternalError.php have inconsistent indents. Please check your reformat settings.